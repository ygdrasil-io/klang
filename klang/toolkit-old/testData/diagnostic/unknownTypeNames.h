some_non_existing_type fun();

bool b;

boolean c;

int single_int;

int int double_int;
