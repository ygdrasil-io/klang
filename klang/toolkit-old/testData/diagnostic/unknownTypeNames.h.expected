unknownTypeNames.h:1: error: unknown type name 'some_non_existing_type'
unknownTypeNames.h:3: error: unknown type name 'bool'
unknownTypeNames.h:5: error: unknown type name 'boolean'
unknownTypeNames.h:9: error: cannot combine with previous 'int' declaration specifier
